core-java-modules/core-java-io-apis-2/src/main/java/com/baeldung/scanner/ScannerNoSuchElementException.java

```diff
@@ -0,0 +1,21 @@
+package com.baeldung.scanner;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+public class ScannerNoSuchElementException {
+
+    public static String readFileV1(String file) throws FileNotFoundException {
```

Comment:
We can rename the parameter to path or pathname which I think is more meaningful.

Reply:
done